README.rst

```diff
@@ -0,0 +1,30 @@
+momkhulu
+=============================
+
+This is a hospital resource allocation tool. Provides an API which can be
+used with RapidPro. Uses WebSockets for real-time page updates.
```

Comment:
WebSockets or Django Channels? or are they the same thing?

Reply:
So WebSockets is the transport protocol which is used by Django Channels.